src/Controller/Organization/User/UserListController.php

```diff
@@ -24,24 +25,31 @@ class UserListController extends AbstractController
     protected UserRepository $userRepository;
     protected OrganizationRepository $organizationRepository;
     private OrganizationSelectorFormFactory $organizationSelectorFormFactory;
+    private PaginatorInterface $paginator;
 
-    public function __construct(OrganizationRepository $organizationRepository, UserRepository $userRepository, OrganizationSelectorFormFactory $organizationSelectorFormFactory)
+    public function __construct(OrganizationRepository $organizationRepository, UserRepository $userRepository, OrganizationSelectorFormFactory $organizationSelectorFormFactory, PaginatorInterface $paginator)
     {
         $this->userRepository = $userRepository;
         $this->organizationRepository = $organizationRepository;
         $this->organizationSelectorFormFactory = $organizationSelectorFormFactory;
+        $this->paginator = $paginator;
     }
 
     public function __invoke(Request $request, Organization $organization): Response
     {
         /** @var Organization $currentOrganization */
         $currentOrganization = $this->getUser();
+        $users = $this->paginator->paginate(
+            $this->userRepository->findByOrganization($organization),
```

Comment:
Are you sure you should not pass a QueryBuilder here ?

Reply:
Indeed... I verified and it works like that but i don't even know why it works. I changed to ```findByOrganizationAndChildrenQb()``` just to be more consistent with other controllers where pagination is used

Reply:
It works because the paginator can handle an array. But here we WANT to paginate the request as well